src/database-controller/src/.dockerignore

```diff
@@ -0,0 +1,63 @@
+.git
```

Comment:
seems it's the same as `sdk/.dockerignore`, maybe keep one in their parent dir?

Reply:
Done.